README.md

```diff
@@ -26,6 +26,10 @@ It is available in [bioconda](https://anaconda.org/bioconda/bx-python) (recommen
 
 ```conda install -c conda-forge -c bioconda bx-python```
 
+It is available in [Debian](https://tracker.debian.org/pkg/python-bx) and Ubuntu:
```

Comment:
```suggestion
It is available in [Debian](https://tracker.debian.org/pkg/python-bx) and [Ubuntu](https://packages.ubuntu.com/python3-bx):
```

Reply:
Nice, thanks! There also badges now for [![DebianBadge](https://badges.debian.net/badges/debian/unstable/python3-bx/version.svg)](https://packages.debian.org/unstable/python3-bx) and  [![DebianBadge](https://badges.debian.net/badges/ubuntu/xenial/python3-bx/version.svg)](http://packages.ubuntu.com/xenial/python3-bx) ... hm ... The Debian version update was already accepted and should migrate to those buttons any time soon. But Xenial - that is a while ago, right? I admit not to know how to get a backport to Xenial.

Reply:
I'll merge it as is (with the Ubuntu link I've added), if that's OK.